pkgs/development/libraries/libarchive/default.nix

```diff
@@ -37,6 +37,22 @@ stdenv.mkDerivation rec {
     hash = "sha256-G4wL5DDbX0FqaA4cnOlVLZ25ObN8dNsRtxyas29tpDA=";
   };
 
+  postPatch = ''
```

Comment:
Why not to put this at the same place of old `postPatch`?

Reply:
I think it's generally best to have attributes defined in the approximate order of the phases, so patch-related attributes come after `src`. When rebasing this to staging I originally didn't even spot that there was already a `postPatch` down there.

Reply:
My rationale is different: first comes the downloadable sources (sources, patches, docs etc.), next the build inputs list and after them all the "transformative" part - unpack, patch, configure, etc.

Reply:
I'm fine either way -- I like the steps in logical order, but I also understand the desire to keep patches simpler.